tox.ini

```diff
@@ -19,7 +19,8 @@ passenv =
     TRAVIS_EVENT_TYPE
 commands=
     core: py.test {posargs:tests/core/}
-    trinity: py.test {posargs:tests/trinity/core/}
+    trinity: py.test {posargs:tests/trinity/core/ --ignore tests/trinity/core/sharding/test_smc_service.py}
+    smc: py.test {posargs:tests/trinity/code/sharding/test_smc_service.py}
```

Comment:
This feels a little brittle, for example if we want to add another SMC test file. If it's worth running a separate SMC job (how long does it take to run?), maybe we can put it in its own test folder, like:

```py
    trinity: py.test {posargs:tests/trinity/core/}
    smc: py.test {posargs:tests/trinity/smc/}
```